cmd/spiffe-helper/config/config.go

```diff
@@ -15,28 +15,35 @@ import (
 	"github.com/spiffe/spiffe-helper/pkg/sidecar"
 )
 
+const (
+	DaemonModeFlagName = "daemon-mode"
+)
+
 const (
 	defaultAgentAddress      = "/tmp/spire-agent/public/api.sock"
 	defaultCertFileMode      = 0644
 	defaultKeyFileMode       = 0600
 	defaultJWTBundleFileMode = 0600
 	defaultJWTSVIDFileMode   = 0600
+	defaultHealthCheckPort   = 8081
+	defaultHealthCheckPath   = "/healthz"
```

Comment:
Should this be `/healthz`? Or `/health`?

Reply:
k8s uses /healthz

Reply:
I see /live and /ready in spire. Do we need those as well?

Reply:
Would it serve up the same content? Or would those serve another function?

Reply:
@keeganwitt discussed this with @kfox1111 offline. There are some differences but we can add those through a follow on pr if there is a need.

Reply:
I guess my question on this then, is do we want to follow spire's convention or k8s's convention?

/health or /healthz?

Reply:
You have to tell k8s in the probe config what the path is. So, it can be either.

Reply:
Seem like healthz is what people will expect so we should go with that.